kpatch-build/kpatch-elf.c

```diff
@@ -76,6 +76,31 @@ int is_debug_section(struct section *sec)
 	       !strncmp(name, ".eh_frame", 9);
 }
 
+#ifdef __powerpc__
+/*
+ * Number of instructions between global and local entry point of a
+ * function is mostly 8 instructions. i.e st_other == 3. The count of
+ * instruction can be decoded only for local function symbols.
+ */
+unsigned int has_local_entry_offset(GElf_Sym sym)
+{
+	return !!PPC64_LOCAL_ENTRY_OFFSET(sym.st_other);
+}
```

Comment:
Also, a couple of changes to make the function consistent with other code:

- make the argument `struct symbol *`
- make the return code `int`
- if the function isn't used outside of the file, make it `static`.

And similar comments for `is_local_func()` below.